ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -61,6 +61,30 @@ export default class ConfirmAddSuggestedToken extends Component {
     const hasTokenDuplicates = this.checkTokenDuplicates(pendingTokens, tokens);
     const reusesName = this.checkNameReuse(pendingTokens, tokens);
 
+    const addTokens = async () => {
+      const tokenEntries = Object.entries(pendingTokens);
+      for(let tokenEntry of tokenEntries) {
```

Comment:
I think this will cause a linting issue, that lack of space after `for`

Reply:
oh yeah....I didn't commit after linting...my bad